tempto-core/src/main/java/com/teradata/tempto/internal/query/QueryResultValueComparator.java

```diff
@@ -238,14 +249,14 @@ private int timestampEqual(Object actual, Object expected)
         return -1;
     }
 
-    private static boolean isIntegerValue(Object value)
+    private static boolean isLongOrNarrower(Object value)
     {
-        return (value instanceof Long || value instanceof Integer || value instanceof Short || value instanceof Byte);
+        return (value instanceof Long || value instanceof Short || value instanceof Byte);
```

Comment:
Why `Integer` is excluded?
`LongOrNarrower` should contain all integers, should it?